src/setup/mouse.c

```diff
@@ -204,7 +210,7 @@ void ConfigMouse(TXT_UNCAST_ARG(widget), void *user_data)
     TXT_SetWindowAction(window, TXT_HORIZ_CENTER, TestConfigAction());
     TXT_SetWindowHelpURL(window, WINDOW_HELP_URL);
 
-    if (gamemission == doom || gamemission == heretic || gamemission == hexen) // [crispy]
+    if (gamemission == doom || gamemission == heretic || gamemission == hexen || gamemission == strife) // [crispy]
```

Comment:
This check has become no-op.

Reply:
Fixed by https://github.com/fabiangreffrath/crispy-doom/pull/998/commits/90f112349863ab2944fbb8cfaa4d749daf9c6294.